ODIN_II/SRC/odin_ii.cpp

```diff
@@ -68,6 +72,18 @@
 #include "GenericReader.hpp"
 #include "BLIF.hpp"
 
+
+#ifdef WINDOWS
```

Comment:
These macros are already added in odin_util, why didn't you use them? This project has a whole utility function, you would need to utilize them rather than writing everything from scratch